modules/gsp-cluster/grafana.tf

```diff
@@ -1,7 +1,25 @@
+data "aws_iam_policy_document" "trust_grafana" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRoleWithWebIdentity"]
+
+    principals {
+      type        = "Federated"
+      identifiers = [module.k8s-cluster.oidc_provider_arn]
+    }
+
+    condition {
+      test = "StringEquals"
+      variable = "OIDC_PROVIDER:sub"
```

Comment:
this `OIDC_PROVIDER:sub` bit doesn't look quite right... think it needs the oidc url in it:

```
"${replace(module.k8s-cluster.oidc_provider_url, "https://", "")}:sub"
```

Reply:
oops!